src/Listener/ToolbarListener.php

```diff
@@ -217,7 +217,7 @@ protected function renderEntries(ProfilerEvent $event)
      * @param  string $currentVersion
      * @return array
      */
-    protected function getLatestVersion($currentVersion)
+    protected function getLatestVersion(string $currentVersion): array
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!